requirements.txt

```diff
@@ -1,5 +1,5 @@
-qiskit>=0.23.1
-pennylane>=0.13.0
+qiskit>=0.23.4
+pennylane>=0.14.0
```

Comment:
It might make sense to add these to `setup.py` as well, so that users that `pip install` this plugin get upgrades of PL and Qiskit as well.

Reply:
Just added that